frontend/src/scenes/insights/InsightContainer.tsx

```diff
@@ -175,17 +176,40 @@ export function InsightContainer(): JSX.Element {
                         align="middle"
                         justify="space-between"
                     >
-                        <Col>
-                            <FunnelCanvasLabel />
-                            <PathCanvasLabel />
-                        </Col>
-                        {lastRefresh && <ComputationTimeWithRefresh />}
+                        {activeView === InsightType.FUNNELS ||
+                            (activeView === InsightType.PATHS && (
+                                <Col>
+                                    <FunnelCanvasLabel />
```

Comment:
We have a double check for `activeView` in the component, we should avoid this for easier maintenance